From c31c7de56e402649053c447ceb20d3e13074adaf Mon Sep 17 00:00:00 2001
From: Nolen Johnson <johnsonnolen@gmail.com>
Date: Tue, 27 Feb 2018 23:29:34 -0500
Subject: [PATCH 115/249] wiki: jfltexx: Update Custom TWRP

* This is temporary until TWRP rekick's off our official build to include the /misc/ patch, which was missed.
* The image functions and OTA is tested on jfltxx, and jfltetmo (which is supported by jfltexx builds).

Change-Id: I1e737193bca66e53aa668773b70bd292c75f9060
---
 _data/devices/jfltexx.yml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/_data/devices/jfltexx.yml b/_data/devices/jfltexx.yml
index f129933..1b6b35c 100644
--- a/_data/devices/jfltexx.yml
+++ b/_data/devices/jfltexx.yml
@@ -11,6 +11,7 @@ cpu: Krait 300
 cpu_cores: 4
 cpu_freq: 1.9GHz
 current_branch: 14.1
+custom_twrp_link: https://www.androidfilehost.com/?fid=962187416754463329
 depth: 7.9mm (0.31 in)
 download_boot: With the device powered off, hold <kbd>Home</kbd> + <kbd>Volume Down</kbd> + <kbd>Power</kbd>.
 gpu: Adreno 320
-- 
2.17.1

